schemas/yaml/Language.yml

```diff
@@ -1,5 +1,5 @@
 $schema: 'http://json-schema.org/draft-07/schema#'
-$id:     'http://schemas.digitallinguistics.io/Language-9.0.0.json'
+$id:     'http://schemas.digitallinguistics.io/Language-9.1.0.json'
```

Comment:
I changed the PR steps so that you don't have to worry about versioning anymore. I'll handle that when it's time to make a release now. (I realized what had happened before is that some schemas got incremented multiple times between releases, when they really only need to be incremented upon release.)